notebook2/notebook/notebook.py

```diff
@@ -377,7 +359,7 @@ def notebook_post():
     safe_user_id = user_id_transform(session['user']['id'])
 
     pod = start_pod(jupyter_token, WORKER_IMAGES[image], name, safe_user_id)
-    session['notebook'] = marshall_notebooks([pod], svc_status = "Running")[0]
+    session['notebook'] = notebooks_for_ui([pod], svc_status = "Running")[0]
```

Comment:
this is wrong if a pod is deleting, right? Either way we should just call `pod_to_ui_dict` directly

Reply:
Pod is never deleting at creation

Reply:
Also, this is going away